release notes/v1.0.0-rc1.md

```diff
@@ -0,0 +1,228 @@
+k6 `v1.0.0-rc1` is here 🎉!
+
+This release marks a special, non-conventional milestone in the k6 software life-cycle, serving as a preview of the upcoming version 1.0.0.
+
+The purpose of this release is to give the community a chance to test the new version, identify any potential issues, and test migrations of any parts affected by breaking changes. If you encounter any problems or have trouble with the migration, we encourage you to report them by creating an issue. Your feedback will help improve the final release. If no critical issues are reported, we plan to release the final `v1.0.0` within the next month.
+
+Here’s a glimpse of what’s new in this release:
+
+- `k6/experimental/webcrypto` promoted to stable and available globally under `crypto`.
+- A revamped end-of-test summary aiming to bring an easier way to understand test results.
+- `k6/browser` provided an API for tracking network requests and responses.
+- The new `k6/secrets` module for retrieving secrets with extension support.
+
+## Breaking changes
+
+- [#4541](https://github.com/grafana/k6/pull/4541) Commas(`,`) are now supported in the values of the `--tag` CLI flag. This is a breaking change, as previously, a comma meant the start of a new set of tag-values. As a comma is a valid symbol for the value of a tag, this is necessary to have equivalency between different ways of setting tags. This still allows multiple tags to be set on the CLI with multiple `--tag key=value` arguments.
+
+### A new default path for the configuration file [#4301](https://github.com/grafana/k6/pull/4301)
+
+When running the `k6 cloud login` or the deprecated `k6 login` commands, a configuration file was automatically created at `{USER_CONFIG_DIR}/loadimpact/config.json`. Now, the configuration file is created at `{USER_CONFIG_DIR}/k6/config.json`.
+
+To migrate your configuration file to the new path:
+
+1. Run `k6 cloud login` or `k6 login` to automatically migrate the configuration file to the new location.
+2. Run `k6 cloud run` or `k6 run` to verify that the version is now fully functional and no related warning is emitted.
+
+The configuration file in the old path remains available and can continue to be used with the previous k6 versions. If you're not using an old version of k6 anymore, consider deleting the files manually.
+
+The `k6 run` commands search for the configuration file in the new location. If it can't find it, it tries to fall back on the old path and then logs a warning message suggesting to migrate it.
+
+## New features
+
+### A revamped end-of-test summary aiming to bring an easier way to understand test results [#4089](https://github.com/grafana/k6/pull/4089), [#4649](https://github.com/grafana/k6/pull/4649)
```

Comment:
@heitortsergent finally we moved this section within `New features`, because a recent change -[#4649](https://github.com/grafana/k6/pull/4649)- made it no longer being a breaking change.

I applied most of your suggestions, but of course a few no longer apply. Note that I used the style you suggested for the "note", but now I moved it to the end of the section, as it is way less relevant now.

Reply:
Sounds good, thanks for letting me know. 🙇